apps/emqx_gateway/src/mqttsn/emqx_sn_impl.erl

```diff
@@ -19,6 +19,14 @@
 
 -behaviour(emqx_gateway_impl).
 
+-include_lib("emqx/include/logger.hrl").
+
+-import(emqx_gateway_utils,
```

Comment:
nit: don't import

Reply:
Copy that. I will fix them in the next PR